### ISSUE TITLE
Implement a way to calculate deviation of measurements.

### ISSUE THREAD

**Issue by qnikst** [tweag/hyperion#39]
Currently hyperion returns only raw measurement data and without external tool it's hard to make sense of data because we don't know how accurate those measurements were. It would be nice if we can measure deviation of the measurements.

**Comment by mboes** [comment]
Issue not actionable as it stands. "Deviation of the measurements" - do you mean variance? Which quantitative measures (moments) specifically do you want to see?

**Comment by qnikst** [comment]
Yes I mean variance,  if hyperion is run without `--raw` flag then it's not possible to understand variance, of the values from report:
```
{"cycles":null
,"measurements":null
,"garbage_collections":null
,"time_in_nanos":839.5436263736264
,"bench_name":"944"
,"alloc":null,"bench_params":[]}],"metadata":{"location":null,"timestamp":"2017-09-05T11:47:47.633059736Z"}}
```

P.S. In case of the `--raw` flag there is enough information in order to implement that test in 3rd party application, but for human readable report this information is missing.
